corehq/apps/geospatial/tasks.py

```diff
@@ -9,3 +28,38 @@ def geo_cases_reassignment_update_owners(domain, case_owner_updates_dict, task_k
     finally:
         celery_task_tracker = CeleryTaskTracker(task_key)
         celery_task_tracker.mark_completed()
+
+
+@serial_task('async-index-es-docs', timeout=30 * 60, queue='background_queue', ignore_result=True)
```

Comment:
I am thinking here if 30 mins timeout is enough for processing all cases considering the max limit of 1 million cases ?
Could consider it to increase to 60 mins just to be on the safer side.
I am curious if we have done any testing in staging for high no. of cases ?

Reply:
(reviewing further, I understand we are planning load testing with QA :+1: )

Reply:
Based on the initial benchmarking done during the tech spec, it takes about 8 minutes to process 300k docs. This means close to 1m _should_ be about ~27 minutes. I agree it might be wise to be a bit more lenient, even with loadtesting pending, so I'm happy to push this up to an hour.

Reply:
Increased the timeout in [707a97d](https://github.com/dimagi/commcare-hq/pull/35126/commits/707a97dcee09d4e3dfd88e7bf5dc6cb7bc0f9bc3).